command/harness/dlite/handler.go

```diff
@@ -0,0 +1,43 @@
+package dlite
+
+import (
+	"io"
+	"net/http"
+
+	"github.com/drone-runners/drone-runner-aws/command/harness"
+	"github.com/go-chi/chi"
+	"github.com/wings-software/dlite/client"
+	"github.com/wings-software/dlite/poller"
+)
+
+func Handler(p *poller.Poller) http.Handler {
+	r := chi.NewRouter()
+	r.Use(harness.Middleware)
+
+	r.Mount("/maintenance_mode", func() http.Handler {
```

Comment:
Add get on maintenance mode to print the current state whether it is in PROD mode or maintenance mode